From 2de968a4c28aaeaf002a7f4cd24a11369d944a8b Mon Sep 17 00:00:00 2001
From: Stefan Agner <stefan.agner@toradex.com>
Date: Thu, 10 Oct 2019 16:15:03 +0200
Subject: [PATCH] apalis-imx8/x/colibri-imx8qxp: enable device tree overlay
 support

Enable device tree overlay support which is used in the current dtconf
implementation in TorizonCore.

Signed-off-by: Stefan Agner <stefan.agner@toradex.com>
---
 configs/apalis-imx8_defconfig     | 1 +
 configs/apalis-imx8x_defconfig    | 1 +
 configs/colibri-imx8qxp_defconfig | 1 +
 3 files changed, 3 insertions(+)

diff --git a/configs/apalis-imx8_defconfig b/configs/apalis-imx8_defconfig
index 2fbb50492f..e8a8c8b8c5 100644
--- a/configs/apalis-imx8_defconfig
+++ b/configs/apalis-imx8_defconfig
@@ -64,4 +64,5 @@ CONFIG_USB_GADGET_PRODUCT_NUM=0xa4a5
 CONFIG_CI_UDC=y
 CONFIG_VIDEO=y
 CONFIG_VIDEO_IMX_HDP_LOAD=y
+CONFIG_OF_LIBFDT_OVERLAY=y
 # CONFIG_EFI_LOADER is not set
diff --git a/configs/colibri-imx8qxp_defconfig b/configs/colibri-imx8qxp_defconfig
index 217e16cb1f..f8882fe25a 100644
--- a/configs/colibri-imx8qxp_defconfig
+++ b/configs/colibri-imx8qxp_defconfig
@@ -62,4 +62,5 @@ CONFIG_USB_GADGET_VENDOR_NUM=0x0525
 CONFIG_USB_GADGET_PRODUCT_NUM=0xa4a5
 CONFIG_CI_UDC=y
 CONFIG_VIDEO=y
+CONFIG_OF_LIBFDT_OVERLAY=y
 # CONFIG_EFI_LOADER is not set
-- 
2.23.0

